mapApp/models/point.py

```diff
@@ -104,9 +104,10 @@ class Point(models.Model):
 
     details = models.TextField(
         _('Please give a brief description of the incident'),
-        max_length=300,
-        blank=True,
-        null=True
+        max_length=500,
+        default=None,
+        blank=False,
+        null=False
```

Comment:
Making this a required field will break form submission from the old version of the app. Probably have to figure out some other way of validating this field in the interim.

Reply:
Added client-side validation using JS.